src/llms/ollama_client.py

```diff
@@ -0,0 +1,45 @@
+from typing import Any, Dict
+
+import ollama
+
+from .base_client import BaseClient
+
+
+class OllamaClient(BaseClient):
+    def __init__(self, model: str):
+        self.model = model
+        super().__init__()
+
+    def chat_completion(self, system: str, user: str, **kwargs) -> str:
+        """
+        Generate a chat completion response using the specified model.
+
+        This function interacts with the Ollama API to generate a response
+        based on input messages from the system and the user. Additional
+        parameters can be customized via kwargs.
+
+        Args:
+            system (str): The content of the message from the system.
+            user (str): The content of the message from the user.
+            **kwargs: Optional keyword arguments for additional options:
+                - num_ctx (int): The context size for the model, default is 4096.
+                - temperature (float): The randomness in the model's output,
+                  default is 0.
+
+        Returns:
+            str: The content of the generated message response.
+        """
+
+        response = ollama.chat(
+            model=self.model,
+            messages=[
+                {"role": "system", "content": system},
+                {"role": "user", "content": user},
+            ],
+            options={
```

Comment:
`options = kwargs`

Reply:
No, it doesn't pass the type checker. I need to explicitly ignore it warning. And isn't that kinda risky if you don't check what `key` is in it?

Reply:
if we use `kwargs.get` kinda lose the purpose of using **kwargs. your call.

Reply:
Yeah, I think this options-passing should be redesigned later... My point is to have a flexible interface to pass in relevant options for different llm providers.